src/main/java/com/wsu/workoutservice/repository/RunnerRepository.java

```diff
@@ -0,0 +1,23 @@
+package com.wsu.workoutservice.repository;
+
+import com.wsu.workoutservice.model.Runner;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface RunnerRepository extends JpaRepository<Runner, Integer> {
+
+    @Query(nativeQuery = true, value = 
+    //"Select * From WorkoutDB.Runner"
+    "Select r.runner_id AS id, r.email as email, r.first_name AS firstName, r.last_name AS lastName, "
+            + "r.gender AS gender, r.height AS height, r.age AS age, r.weight AS weight "
+            + "FROM Runner r WHERE :search IS NULL OR (r.first_name = :search)"

```

Comment:
It would be nice to have a filter for the runner email as well since multiple runners can have the same first name. 
Replace this:
`(r.first_name = :search)` with `(r.first_name = :search OR r.email = :search)`